index.d.ts

```diff
@@ -6,6 +6,13 @@ declare namespace camelcase {
 		@default false
 		*/
 		readonly pascalCase?: boolean;
+
+		/**
+		Save sequence capital chars: `foo-BAR` → `fooBAR`
+
```

Comment:
Would be useful to include some use-cases for this option. It's not always clear to users when they would want to use it.